reverse_engineering/hqlToCollectionsVisitor.js

```diff
@@ -649,10 +642,6 @@ class Visitor extends HiveParserVisitor {
 		}
 	}
 
-	visitAlterStatementSuffixRename(ctx) {
```

Comment:
deleted the wrong one 🙂 the last one declared takes precedence afaik, so this particular one was in use